.gitignore

```diff
@@ -114,3 +114,8 @@ data/catch-ignore.yml
 release_config.json
 web/userdata.js
 location.json
+
+#Multiple config
+configs/
+!configs/config.json.example
+!configs/release_config.json.example
```

Comment:
Oooops! I thought I had moved those two to the configs folder for his PR. If I did that could we move forward with this PR?